Sources/Compiler/Parse/ParserState.swift

```diff
@@ -135,6 +138,20 @@ struct ParserState {
     return token
   }
 
+  /// Returns whether the next token has the specified kind, or `false` if the stream is empty.
```

Comment:
```suggestion
  /// Returns whether a token of the given `kind` is next in the input.
```

"specified" is just a 3-syllable way to say “given” and even that should usually be avoided.

[And it seems I'll never be comfortable reading “Returns whether” in documentation.  Grumpy old man!]